src/load-data.js

```diff
@@ -1,5 +1,29 @@
 'use strict'
 
+const TILL_KEY = 'SPLITIO.splits.till'
+
 export default function loadDataIntoLocalStorage ({ serializedData = {} }) {
-  throw new Error('Not implemented')
+  const { segmentsData, since, splitsData, userId, usingSegmentsCount } = serializedData
+  const windowLocalStorage = window.localStorage
+
+  // Do not load data if current localStorage data is more recent
+  if (since <= windowLocalStorage.getItem(TILL_KEY)) {
+    return
+  }
+  windowLocalStorage.clear()
```

Comment:
Split.IO recommends doing this. I am not sure if this will have negatively affect partners that will run this script.